src/app/pages/AccountDetailsPage/AccountNFTCollectionCard.tsx

```diff
@@ -0,0 +1,158 @@
+              <CopyToClipboard value={ethContractAddress} />
+            </Box>
+          }
+          disableTypography
+          title={
+            <Box sx={{ display: 'flex' }} gap={4}>
+              <Breadcrumbs separator="›" aria-label="breadcrumb">
+                <Typography fontSize={24}>
+                  <Link
+                    preventScrollReset={true}
+                    component={RouterLink}
+                    to={RouteUtils.getAccountTokensRoute(scope, address, 'ERC721', '')}
+                  >
+                    {t('nft.accountCollection')}
+                  </Link>
+                </Typography>
+                {isFetched && (
+                  <Box sx={{ display: 'flex', alignItems: 'baseline' }} gap={2}>
+                    <Typography color={COLORS.brandExtraDark} fontSize={24}>
+                      {inventory?.length && inventory?.[0].token.name
+                        ? inventory?.[0].token.name
+                        : t('common.collection')}
+                    </Typography>
+                    {!!totalCount && (
+                      <Typography>({`${isTotalCountClipped ? ' > ' : ''}${totalCount}`})</Typography>
+                    )}
+                  </Box>
+                )}
+              </Breadcrumbs>
+              {isLoading && <Skeleton variant="text" sx={{ width: '50%' }} />}
+            </Box>
+          }
+        />
+      </LinkableDiv>
+      <CardContent>
+        <ErrorBoundary light={true}>
+          <AccountNFTCollection
+            inventory={inventory}
+            isFetched={isFetched}
+            isLoading={isLoading}
+            totalCount={totalCount}
+            isTotalCountClipped={isTotalCountClipped}
+            pagination={pagination}
+            scope={scope}
+          />
+        </ErrorBoundary>
+      </CardContent>
+    </Card>
+  )
+}
+
+type AccountNFTCollectionProps = {
+  inventory: EvmNft[] | undefined
+  isLoading: boolean
+  isFetched: boolean
+  isTotalCountClipped: boolean | undefined
+  totalCount: number | undefined
+  pagination: {
+    rowsPerPage: number
+    selectedPage: number
+    linkToPage: (pageNumber: number) => To
+  }
+  scope: SearchScope
+}
+
+const AccountNFTCollection: FC<AccountNFTCollectionProps> = ({
+  inventory,
+  isLoading,
+  isFetched,
+  isTotalCountClipped,
+  pagination,
+  scope,
+  totalCount,
+}) => {
+  const { t } = useTranslation()
+
+  return (
+    <>
+      {isLoading && <Skeleton variant="rectangular" sx={{ height: 200 }} />}
+      {isFetched && !totalCount && <CardEmptyState label={t('tokens.emptyInventory')} />}
+      {!!inventory?.length && (
+        <>
+          <ImageList gap={10}>
+            {inventory?.map(instance => {
+              const to = RouteUtils.getNFTInstanceRoute(scope, instance.token?.contract_addr, instance.id)
+              return (
+                <ImageListItem key={instance.id}>
+                  <ImageListItemImage instance={instance} to={to} />
```

Comment:
Can we actually unset the fixed height, width of the image here. And make it display as a cover -> `object-fit: cover;` It butchers the image ration

Reply:
Prod Nexus API which deployed version is using is missing a lot of data. To see (at least some) images you need to switch to staging Nexus locally.

Sure, I will check object-fit now.